website/src/docs/usage.md

```diff
@@ -65,18 +62,18 @@ will be brought up.
 
 If you call Task with the `--global` (alias `-g`) flag, it will look for your
 home directory instead of your working directory. In short, Task will look for a
-Taskfile that matches `$HOME/{T,t}askfile.{yml,yaml}` .
+Taskfile that matches `$HOME/`<code>{T,t}askfile.{yml,yaml}</code> .
```

Comment:
This one too.

They still render okay but we should still revert those changes.

And there are other like those.